packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -215,6 +215,16 @@ export namespace CommonCommands {
         category: VIEW_CATEGORY,
         label: 'Toggle Bottom Panel'
     };
+    export const PIN_TAB: Command = {
+        id: 'core.pin.tab',
+        category: VIEW_CATEGORY,
+        label: 'Pin Tab'
+    };
+    export const UNPIN_TAB: Command = {
+        id: 'core.unpin.tab',
+        category: VIEW_CATEGORY,
+        label: 'Unpin Tab'
+    };
```

Comment:
It looks like the VSCode ID's for these are `workbench.action.pinEditor`  and `workbench.action.unpinEditor`. We should either use those names or [map those names](https://github.com/eclipse-theia/theia/blob/master/packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts) to this so that this command can be run by plugins.